### PR TITLE
Here is a new version.

### PR DESCRIPTION
**Pull request by Waruggy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,11 +26,19 @@ In this project you will create a login page and request a token from the server
...
+
+- [x] What steps can you take in your web apps to keep your data secure?
+
+Https requests and requiring sign in for important data.
+
+- [x] Describe how web servers work.
+
+- [x] Which HTTP methods can be mapped to the CRUD acronym that we use when interfacing with APIs/Servers.
 
+create=post, read=get, update=put, and delete=delete.
```
Comment (srsimps19) [inline README.md:41]: Good answer!

File: client/src/App.js [hunk 2]
```diff
@@ -1,19 +1,27 @@
 import React, { useState } from "react";
-import { BrowserRouter as Router, Route } from "react-router-dom";
+import { BrowserRouter as Router, Route, Redirect } from "react-router-dom";
 
 import Login from "./components/Login";
 import "./styles.scss";
+import BubblePage from "./components/BubblePage"
+
+const PrivateRoute = ({component: Component, ...rest}) => {
```
Comment (srsimps19) [inline client/src/App.js:8]: I would love to see this in a separate component

File: client/src/components/ColorList.js [hunk 3]
```diff
@@ -18,13 +18,27 @@ const ColorList = ({ colors, updateColors }) => {
 
   const saveEdit = e => {
     e.preventDefault();
-    // Make a put request to save your updated color
-    // think about where will you get the id from...
-    // where is is saved right now?
+    axiosWithAuth().put(`http://localhost:5000/api/colors/${colorToEdit.id}`, colorToEdit)
+    .then(res => {
```
Comment (srsimps19) [inline client/src/components/ColorList.js:22]: noice nested axios call!

**Review by srsimps19 — APPROVED** [review]
Looks great Charles!

**Comment by Waruggy** [comment]
I figured out yarn is not working when I usually use yarn.

**Pull request merged**